lisp/forge-core.el

```diff
@@ -61,6 +61,8 @@
      "salsa.debian.org" forge-gitlab-repository)
     ("framagit.org" "framagit.org/api/v4"
      "framagit.org" forge-gitlab-repository)
+    ("gitlab.gnome.org" "gitlab.gnome.org/api/v4"
+     "GNOME-GitLab" forge-gitlab-repository)
```

Comment:
So far we have always used the same value for `id` as for `githost`. I think I want to stick to that.

Reply:
Allright, I've changed that accordingly in the recently pushed version of the commit.